scripts/update_proto_files.py

```diff
@@ -0,0 +1,57 @@
+"""
+Creates/updates the proto files in cognite/client/_proto/ using definitions from:
+https://github.com/cognitedata/protobuf-files
+
+Requires `protoc` to be installed. On MacOS, you can install it with Homebrew:
+$ brew install protobuf
+
+Note:
+As long as we support `protobuf >= 4`, we need to use the last `protoc` version with v4 support.
+That seems to be release 25.4, which can be downloaded here:
+https://github.com/protocolbuffers/protobuf/releases
+"""
+
+import os
+import shlex
+import subprocess
+import tempfile
+from pathlib import Path
+
+import requests
+
+URL_BASE = "https://raw.githubusercontent.com/cognitedata/protobuf-files/master/v1/timeseries/"
+FILES = "data_point_list_response.proto", "data_points.proto", "data_point_insertion_request.proto"
+PROTO_DIR = str(Path("cognite/client/_proto").resolve())
+
+
+def download_proto_files_and_compile():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        os.chdir(tmpdir)
+        for file in map(Path, FILES):
+            file.touch()
+            file.write_bytes(requests.get(f"{URL_BASE}{file}").content)
+        protoc_command = " ".join(("protoc", *FILES, f"--python_out={PROTO_DIR}", f"--pyi_out={PROTO_DIR}"))
+        subprocess.run(shlex.split(protoc_command), check=True)
```

Comment:
It is a bit weird to me first join the command parameters, and then split it again using shlex.
```patch
diff --git a/scripts/update_proto_files.py b/scripts/update_proto_files.py
index 2ce4e724..c65f4422 100644
--- a/scripts/update_proto_files.py
+++ b/scripts/update_proto_files.py
@@ -20,7 +20,7 @@ from pathlib import Path
 import requests
 
 URL_BASE = "https://raw.githubusercontent.com/cognitedata/protobuf-files/master/v1/timeseries/"
-FILES = "data_point_list_response.proto", "data_points.proto", "data_point_insertion_request.proto"
+FILES = ["data_point_list_response.proto", "data_points.proto", "data_point_insertion_request.proto"]
 PROTO_DIR = str(Path("cognite/client/_proto").resolve())
 
 
@@ -30,8 +30,8 @@ def download_proto_files_and_compile():
         for file in map(Path, FILES):
             file.touch()
             file.write_bytes(requests.get(f"{URL_BASE}{file}").content)
-        protoc_command = " ".join(("protoc", *FILES, f"--python_out={PROTO_DIR}", f"--pyi_out={PROTO_DIR}"))
-        subprocess.run(shlex.split(protoc_command), check=True)
+        protoc_command = ["protoc"] + FILES + [f"--python_out={PROTO_DIR}", f"--pyi_out={PROTO_DIR}"]
+        subprocess.run(protoc_command, check=True)
 
 
 def patch_bad_imports():
```

But this is just a nit.